Scripts/Swift/Helpers/SafeShell.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+@discardableResult
+func safeShell(_ command: String) throws -> String {
+    let task = Process()
+    let pipe = Pipe()
+    
+    task.standardOutput = pipe
+    task.standardError = pipe
+    task.arguments = ["-c", command]
+    task.executableURL = URL(fileURLWithPath: "/bin/zsh")
+    task.standardInput = nil
+
+    try task.run()
+    
+    let data = pipe.fileHandleForReading.readDataToEndOfFile()
+    let output = String(data: data, encoding: .utf8)!
```

Comment:
can we avoid force unwrapping here and return a default string in case of nil?

Reply:
Good suggestion. Let me try.